extensions/httpserver/internal.m

```diff
@@ -37,7 +37,7 @@ @interface HSHTTPServer : HTTPServer
 
 @interface HSHTTPDataResponse : HTTPDataResponse
 @property int hsStatus;
-@property (nonatomic, copy) NSMutableDictionary *hsHeaders;
+@property (nonatomic, copy) NSDictionary *hsHeaders;
```

Comment:
The `copy` property implies an immutable copy will be made, so marking the property as `NSMutableDictionary` makes no sense.

Reply:
no objections